openapi-contracts/src/main/com/sinch/sdk/domains/numbers/models/v1/errors/InternalErrorImpl.java

```diff
@@ -62,7 +70,9 @@ public String getMessage() {
   }
 
   public OptionalValue<String> message() {
-    return null != error ? error.map(InternalErrorError::getMessage) : OptionalValue.empty();
+    return null != error && error.isPresent()
+        ? error.map(f -> ((InternalErrorErrorImpl) f).message()).orElse(OptionalValue.empty())
```

Comment:
What is the reason why you now use the implementation instead of the interface to reach the property value?